tests/h_item/h_item_api.cpp

```diff
@@ -56,94 +112,203 @@ class TEST_NAME : public util::test_base {
       auto testQueue = util::get_cts_object::queue();
 
       const auto kernelGroupRange =
-          util::get_cts_object::range<numDims>::get(8, 4, 2);
+          util::get_cts_object::range<numDims>::get(2, 2, 2);
```

Comment:
I wonder whether having the same value everywhere could not hide some bugs with an implemention getting confused about x/y/z ordering...

Reply:
Thanks. Done by e16addc